src/tokenizer/stop_word_filter.rs

```diff
@@ -98,3 +98,15 @@ where
         false
     }
 }
+
+impl Default for StopWordFilter {
+    fn default() -> StopWordFilter {
+        let words = vec![
+            "a", "an", "and", "are", "as", "at", "be", "but", "by", "for", "if", "in", "into",
```

Comment:
this could be a `[&'static str; N]`.

Reply:
Oh nice. that should be "leaner" rather than building out a vec.